<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>821</width>
      <height>531</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="done">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>800</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Done</string>
     </property>
    </widget>
    <widget class="QLabel" name="display_screen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>4</y>
       <width>821</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview Camera</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter IP Address Camera </string>
    </property>
   </widget>
   <widget class="QTextEdit" name="ipAddress">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>681</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="check_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: green;color: white</string>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>670</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_camera">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>670</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #FFCC33; color :white</string>
    </property>
    <property name="text">
     <string>Add Camera</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
